flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -1224,4 +1230,26 @@ private void getActiveNotifications(Result result) {
             result.error(GET_ACTIVE_NOTIFICATIONS_ERROR_CODE, e.getMessage(), e.getStackTrace());
         }
     }
+    private void getNotificationChannels(Result result) {
+        if (VERSION.SDK_INT < VERSION_CODES.O) {
+            result.error(GET_NOTIFICATION_CHANNELS_ERROR_CODE, GET_NOTIFICATION_CHANNELS_ERROR_MESSAGE, null);
```

Comment:
Could you change to be consistent with the other APIs available? Currently no exceptions are thrown for other methods around notification channels to be consistent with [NotificationManagerCompat](https://developer.android.com/reference/androidx/core/app/NotificationManagerCompat) as throwing an exception could prevent an app from executing the logic that follows it to show a notification when it shouldn't be a show stopper.

In this case, returning an empty list would make it consistent with the native APIs (see https://developer.android.com/reference/androidx/core/app/NotificationManagerCompat#getNotificationChannels())